src/main/webapp/WEB-INF/tags/instructor/home/coursePanel.tag

```diff
@@ -5,8 +5,8 @@
 <%@ attribute name="courseTable" type="teammates.ui.template.CourseTable" required="true" %>
 <%@ attribute name="index" required="true" %>
 <c:set var="isNotLoaded" value="${empty courseTable.buttons}" />
-<div class="panel panel-primary" id="course-${index}"<c:if test="${isNotLoaded}"> style="cursor: pointer;"</c:if>>
-    <div class="panel-heading">
+<div class="panel panel-primary" id="course-${index}">
+    <div class="panel-heading ajax_auto" <c:if test="${isNotLoaded}"> style="cursor: pointer;"</c:if>>
```

Comment:
Is there any particular reason why you moved the style down?

Also, perhaps you could move the `.ajax_auto` up (and maybe change its name)? I notice you're using a lot of `.parent()` calls in the JS below.